Assignment2/alvaroalvi-fromRDFtoJSON-LD.jsonld

```diff
@@ -1,53 +1,69 @@
 {
-
 	"@context":{
 		"xsd":"http://www.w3.org/2001/XMLSchema#",
-		"pf":"http://www.ontologies.org/ontologies/Class#"
+		"pf":"http://www.ontologies.org/ontologies/Class#",
 		"includes": "rdfs:label",
-		"Measurement8401": "rdfs:label",
-		"hasTemperature": "rdfs:label",
+		"hasMeasurement": "rdfs:label",
+		"hasTemperature":{
+			"rdfs":"label",
+			"@type":"xsd:date"
+		},
 		"hasOwner":"rdfs:label",
-		"atTime": "rdfs:label",
-		"hasName": "rdfs:label",
+		"atTime":{
+			"rdfs":"label",
+			"@type":"xsd:dateTime"
+		},
+		"hasName":{
+			"rdfs":"label",
+			"@type":"xsd:string"
+		}

```

Comment:
Wrong definitions of hasTemperature, atTime and hasName. You should only provide a definition as you did with hasMeasurement. 
In addition, using "rdfs:label" for all the attributes is not a good practice since when you have to query your data you need to refer to a different attribute. Provide definitions like "pf:includes", "pf:hasTemperature", etc.